examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -119,7 +119,7 @@
     keys=['Original', 'SimpleImputer', 'IterativeImputer'], axis=1
 )
 
-# plot boston results
+# plot california housing results
```

Comment:
This is unrelated to the title.

For future reference, it is okay to open another PR for typos.